py-polars/polars/io/database.py

```diff
@@ -151,7 +151,7 @@ def __exit__(
         exc_val: BaseException | None,
         exc_tb: TracebackType | None,
     ) -> None:
-        # iif we created it and are finished with it, we can
+        # if we created it and are finished with it, we can
```

Comment:
iif is actually a valid term and I think it applies here:
https://en.wikipedia.org/wiki/If_and_only_if

Could you revert this one?

Reply:
that's briefly what i thought too but then it should be iff

Reply:
oh you're right.